### ISSUE TITLE
Decide on coding style and enforce mostly with PHPCS

### ISSUE THREAD

**Issue by Krinkle** [perftools/xhgui#299]
Less burden on reviewers, less guess work on the contributing person (ergo me). Also automation in supporting IDEs with automatic fixing/adapting as needed.

It looks like the current code looks mostly like PSR-2.  Is that intentional?

**Comment by markstory** [comment]
> It looks like the current code looks mostly like PSR-2. Is that intentional?

Yes and no. I think PSR2 is a good base target to get to. I never got around to fully updating the existing code though.

**Comment by glensc** [comment]
@markstory perhaps remove Sticker CI integration? it seems broken and no support.

It's marked CI job as failed, example: https://github.com/perftools/xhgui/pull/363

but can't retrieve any details why.

the link from PR leads to https://stickler-ci.com/repositories/156-perftools-xhgui but that says "bummer, The requested address /repositories/156-perftools-xhgui was not found." 

The "login" on the page leads to https://stickler-ci.com/github/authorize, and redirects back to the same page, and even still shows "login" even I've logged in.

I've contacted their support, and they have not even responded over several months.

can you remove the integration? as I think I don't even have permission to remove the integration.

**Comment by markstory** [comment]
@glensc Oddly enough I run stickler, and never got any support emails. I can look into what is not working with the login flow. There are [build logs and results](https://github.com/perftools/xhgui/runs/1321901057) in the Github checks views though.

**Comment by glensc** [comment]
@markstory ok, I can see output via GitHub actions, so I guess can leave as is then.

The other link gave an error and I contacted their support, but the ☎️ is dead

![image](https://user-images.githubusercontent.com/199095/97717902-f969a400-1acd-11eb-8bf1-1b1868cb5626.png)
